tests/pfcwd/test_pfcwd_function.py

```diff
@@ -40,7 +41,31 @@ def stop_pfcwd(duthosts, rand_one_dut_hostname):
     logger.info("--- Stop Pfcwd --")
     duthost.command("pfcwd stop")
 
+
 class PfcCmd(object):
+    buffer_model_initialized = False
+    buffer_model = None
+
+    @staticmethod
+    def isBufferInApplDb(asic):
+        if not PfcCmd.buffer_model_initialized:
+            PfcCmd.buffer_model = asic.run_redis_cmd(
+                argv=[
+                    "redis-cli", "-n", "4", "hget",
+                    "DEVICE_METADATA|localhost", "buffer_model"
+                ]
+            )
```

Comment:
@smaheshm, can we get 'DEVICE_METADATA' info using this asic instance?

Reply:
yes. But the code above seems fine, is there a concern.

Reply:
Thank you @smaheshm. wanted to make sure this works on multi asic as well